tests/test_yaml_loader.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `mllaunchpad.config` module."""
+
+# Stdlib imports
+from unittest import mock
+import yaml
+# Application imports
+import mllaunchpad.yaml_loader as yloader
+
+test_file_yaml = b"""
+    dbms: !include test
+     """
+test = b"""
+    xob10:
+        type: oracle
+    """
+
+
+@mock.patch('builtins.open', new_callable=mock.mock_open, read_data=test_file_yaml)
+def test_yaml_include(mo):
+    """Test include sub config files."""
+    handlers = (mo.return_value, mock.mock_open(read_data=test).return_value,)
```

Comment:
Looks like the error can be solved the same way as with the other tests -- by assigning the file name to the mocked handler. Try this:
```suggestion
    mo.return_value.name = "./foobar.yml"
    mo2 = mock.mock_open(read_data=test)
    mo2.return_value.name = "test"
    handlers = (mo.return_value, mo2.return_value)
```